src/modules/fancyzones/lib/WindowMoveHandler.cpp

```diff
@@ -193,8 +194,8 @@ void WindowMoveHandlerPrivate::MoveSizeStart(HWND window, HMONITOR monitor, POIN
     m_shiftHook->enable();
     m_ctrlHook->enable();
 
-    // This updates m_dragEnabled depending on if the shift key is being held down.
-    UpdateDragState(window);
+    UpdateDragState(); // This updates m_dragEnabled depending on if the shift key is being held down    
+    ElevatedDragCheck(window); // Notifies user if unable to drag elevated window

```

Comment:
Put the comments above the instructions, also remove extra spaces ad the end of the first comment.